Jenkinsfile

```diff
@@ -53,7 +52,7 @@ pipeline {
             stage("Update npm version") {
               steps {
                 sh "mkdir -p src/main/resources"
-                sh "echo '${params.SCHEMA.trim()}' > src/main/resources/schema.graphql"
+                sh "wget -O src/main/resources/schema.graphql https://raw.githubusercontent.com/nationalarchives/tdr-consignment-api/master/schema.graphql"
```

Comment:
This directory path is used twice: `src/main/resources/schema.graphql https://raw.githubusercontent.com/nationalarchives/tdr-consignment-api/master/schema.graphql`. Not sure it could be abstracted out to a variable?

Reply:
Makes sense, I've moved `src/main/resources` into a variable and the github url into another one. This stops the repetition but you can still see what's going on.